src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -2788,7 +2788,7 @@ def test_aks_nodepool_add_with_ossku_windows2022(self, resource_group, resource_
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
     @AllowLargeResponse()
-    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='eastus')
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest')
```

Comment:
Why remove the default location? Better set one (`westus2`), and live test pipeline/cli runner will override the setting if necessary.

Reply:
Requeued a [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=71013955&view=results) for your, the previous one is running all the test cases.